### PR TITLE
feat(stdlibs/std)!: remove `std.CurrentRealmPath`

### PR DESCRIPTION
**Pull request by linhpn99**
<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [x] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

**Comment by linhpn99** [comment]
@thehowl hope u give a review

File: gnovm/stdlibs/std/emit_event_test.go [hunk 1]
```diff
@@ -13,7 +13,11 @@ import (
 
 func TestEmit(t *testing.T) {
 	m := gno.NewMachine("emit", nil)
-	pkgPath := CurrentRealmPath(m)
+
+	elgs := sdk.NewEventLogger()
```
Comment (ltzmaxwell) [inline gnovm/stdlibs/std/emit_event_test.go:17]: seems we don't need this line.

Comment (linhpn99) [reply gnovm/stdlibs/std/emit_event_test.go:17]: It needs a context check below. It will panic if it doesn't pass.

File: docs/reference/stdlibs/std/chain.md [hunk 2]
```diff
@@ -39,19 +39,6 @@ arguments acting as key-value pairs to be included in the emitted event.
 ```go
 std.Emit("MyEvent", "myKey1", "myValue1", "myKey2", "myValue2")
 ```
----
```
Comment (thehowl) [inline docs/reference/stdlibs/std/chain.md:42]: leave one of these separators (`---`)

File: docs/reference/stdlibs/std/testing.md
Comment (thehowl) [inline docs/reference/stdlibs/std/testing.md:1]: why these changes?

File: docs/reference/stdlibs/std/testing.md [hunk 3]
```diff
@@ -5,23 +5,23 @@ id: testing
 # Testing
 
 ```go
-func TestCurrentRealm() string
```
Comment (thehowl) [inline docs/reference/stdlibs/std/testing.md:8]: Can you also remove this function and just use `std.CurrentRealm().PkgPath()`?

See `tests/stdlibs/std`

File: gnovm/tests/files/zrealm_natbind0.gno [hunk 4]
```diff
@@ -12,14 +12,14 @@ func init() {
 }
 
 func main() {
-	// NOTE: this test uses GetHeight and CurrentRealmPath, which are "pure"
+	// NOTE: this test uses GetHeight and CurrentRealm, which are "pure"
 	// natively bound functions (ie. not indirections through a wrapper fn,
 	// to convert the types to builtin go/gno identifiers).
 	f := node.(func() int64)
 	println(f())
-	node = std.CurrentRealmPath
```
Comment (thehowl) [inline gnovm/tests/files/zrealm_natbind0.gno:20]: Mhh, no, here please use `GetChainID` instead.

**Review by thehowl — COMMENTED** [review]
Mostly good

File: gnovm/stdlibs/std/emit_event_test.go (hunk 1 above)
Comment (ltzmaxwell) [reply gnovm/stdlibs/std/emit_event_test.go:17]: I mean like this: 	
m.Context = ExecContext{}

the logger is not actually used by X_getRealm.

File: gnovm/tests/files/zrealm_std3.gno
Comment (thehowl) [inline gnovm/tests/files/zrealm_std3.gno:1]: Why is this commented? Does it not work with `CurrentRealm().PkgPath()`?

File: gnovm/tests/files/zrealm12.gno
Comment (thehowl) [inline gnovm/tests/files/zrealm12.gno:1]: Same here

File: docs/reference/stdlibs/std/testing.md (hunk 3 above)
Comment (thehowl) [reply docs/reference/stdlibs/std/testing.md:8]: Remove this from docs please

**Comment by linhpn99** [comment]
@thehowl does my PR is good enough to approve ?

**Review by thehowl — APPROVED** [review]

**Comment by thehowl** [comment]
LGTM, @gfanton for second review

**Review by gfanton — APPROVED** [review]

**Review by thehowl — APPROVED** [review]
